dags/pull_here.py

```diff
@@ -52,3 +54,21 @@
         bash_command = '/data/airflow/airflow_venv/bin/python3 /data/airflow/data_scripts/here/traffic/here_api.py -d /data/airflow/data_scripts/here/traffic/config.cfg -s {{ yesterday_ds_nodash }} -e {{ yesterday_ds_nodash }} ', 
         dag=dag,
         )
+
+#get dags to trigger from airflow variable 
+DAGS_TO_TRIGGER = Variable.get('here_dag_triggers', deserialize_json=True)
```

Comment:
This counts as top level code which means the variable gets read every ~30s. Instead put Variable.get inside an `@task`: https://www.astronomer.io/docs/learn/dag-best-practices?tab=good-practice#avoid-top-level-code-in-your-dag-file. 
Don't be lazy and convert the DAG to task flow 😉